nuxeo-ai-web-ui/src/main/resources/web/nuxeo.war/ui/nuxeo-ai/admin/admin-ai-export.html

```diff
@@ -0,0 +1,169 @@
+            <paper-input id="split" label="[[i18n('admin.ai.split')]]" value="{{split_prop}}"></paper-input>
+            <paper-button on-tap="_recompute">[[i18n('admin.ai.stats.action')]]</paper-button>
+            <paper-button on-tap="_export">[[i18n('admin.ai.export.action')]]</paper-button>
+        </nuxeo-card>
+        <nuxeo-card heading="[[i18n('admin.ai.stats')]]">
+            <div class="row-container">
+                Total possible: [[_total.numericValue]] &nbsp;
+                Valid count: [[_count.numericValue]]
+            </div>
+            <nuxeo-data-table id="statsResult" items="[[_filter(stats)]]">
+                <nuxeo-data-table-column name="[[i18n('admin.ai.stats.field')]]" flex="10">
+                    <template>
+                        [[item.field]]
+                    </template>
+                </nuxeo-data-table-column>
+                <nuxeo-data-table-column name="[[i18n('admin.ai.stats.type')]]" flex="10">
+                    <template>
+                        [[item.type]]
+                    </template>
+                </nuxeo-data-table-column>
+                <nuxeo-data-table-column name="[[i18n('admin.ai.stats.value')]]" flex="50">
+                    <template>
+                        [[_displayVal(item)]]
+                    </template>
+                </nuxeo-data-table-column>
+            </nuxeo-data-table>
+        </nuxeo-card>
+        <nuxeo-card heading="[[i18n('admin.ai.export')]]">
+            <div class="row-container">
+               [[export]]
+            </div>
+        </nuxeo-card>
+    </template>
+    <script>
+        Polymer({
+            is: 'nuxeo-admin-ai-export',
+            behaviors: [Nuxeo.LayoutBehavior],
+            properties: {
+                in_props: {
+                    type: String,
+                    value: "dc:title,file:content"
+                },
+                out_props: {
+                    type: String,
+                    value: "dc:lastContributor"
+                },
+                split_prop: {
+                    type: String,
+                    value: "75"
+                },
+                query: {
+                    type: String,
+                    value: "SELECT * FROM Document WHERE ecm:primaryType = 'File'"
+                },
+                export: {
+                    type: String,
+                    value: ""
+                },
+                stats: {
+                    type: String
+                },
+                _total: {
+                    type: Object,
+                    computed: '_computeTotal(stats.splices)'
+                },
+                _count: {
+                    type: Object,
+                    computed: '_computeCount(stats.splices)'
+                }
+            },
+
+            _displayVal: function(val) {
+                var toReturn = "";
```

Comment:
Mixing single and double quotes for writing strings. Avoid mixing them. Using single quotes is preferred.